From: Jeff Licquia <licquia@debian.org>
Date: Sat, 21 Dec 2019 22:31:04 +0100
Subject: Build system patch

Forwarded: not-needed
Last-Update: 2011-02-20

This patch makes the build system print useful information in case of
a build failure.
---
 Makedefs.in | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Makedefs.in b/Makedefs.in
index 596e4b0..cb79941 100644
--- a/Makedefs.in
+++ b/Makedefs.in
@@ -82,11 +82,8 @@ WARNINGS	=	@WARNINGS@
 # Rules...
 #
 
-.SILENT:
 .SUFFIXES:	.a .c .cxx .h .o
 .c.o:
-	echo Compiling $<...
 	$(CC) $(CPPFLAGS) $(CFLAGS) -c $<
 .cxx.o:
-	echo Compiling $<...
 	$(CXX) $(CPPFLAGS) $(CXXFLAGS) -c $<
